build.gradle

```diff
@@ -191,6 +191,9 @@ application {
 // see https://pekko.apache.org/docs/pekko/current/additional/packaging.html#gradle-the-jar-task-from-the-java-plugin
 //////////////////////////////////////////////////////////////////////
 tasks.shadowJar {
+  filesMatching('reference.conf') {
+    duplicatesStrategy = DuplicatesStrategy.INCLUDE
+  }
```

Comment:
Do you see any change to cover this somehow by some test or similar?

Reply:
Maybe in CI... One could create a jar and run a short simulation? Maybe a task for a student?